_includes/about.html

```diff
@@ -1,5 +1,14 @@
 {% assign data = site.data.[page.year] %}
 
+<div class="tickets" > 
+  <h1 class="glitch" data-text="Qualifiers are happening now!!!">Qualifiers are happening now!!!</h1> 
+  <p> Join in
+    <b><a target="_blank" href="https://quals.sigint.mx">here</a></b>
+  </p>
+</div>
```

Comment:
This should be wrapped in an if statement that checks if the qual started field is now or you can add a new field `quals live`. You should also make the link configurable too.